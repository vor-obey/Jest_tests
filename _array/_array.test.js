describe('test _array', () => {
  it('should add new element in collection', () => {
    const shoppingList = [
      'diapers',
      'kleenex',
      'trash bags',
      'paper towels',
      'milk',
    ];
    shoppingList.push('beer');

    expect(shoppingList).toContainEqual('beer');
    expect(shoppingList).toHaveLength(6);
    expect(shoppingList).toMatchSnapshot();
  })
})
