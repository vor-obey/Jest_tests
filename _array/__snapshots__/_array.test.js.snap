// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test _array should add new element in collection 1`] = `
Array [
  "diapers",
  "kleenex",
  "trash bags",
  "paper towels",
  "milk",
  "beer",
]
`;
